include/oneapi/dpl/experimental/kt/esimd_radix_sort.h

```diff
@@ -69,8 +74,8 @@ radix_sort_by_key(sycl::queue __q, _KeysIterator __keys_first, _KeysIterator __k
     auto __vals_keep = oneapi::dpl::__ranges::__get_sycl_range<oneapi::dpl::__par_backend_hetero::access_mode::read_write,
                                                           _ValsIterator>();
     auto __vals_rng = __vals_keep(__vals_first, __vals_first + (__keys_last - __keys_first));
-
-    return __impl::__radix_sort_by_key<__is_ascending, __radix_bits>(__q, __keys_rng.all_view(), __vals_rng.all_view(), __param);
+    auto __pack = __impl::__utils::__rng_pack{::std::move(__keys_rng.all_view()), ::std::move(__vals_rng.all_view())};
```

Comment:
I believe `std::move` not required here.

Reply:
I slightly changed the line. Now, I move  `__keys_rng` instead of `__keys_rng.all_view()`, which may already be temporary.